ann_benchmarks/algorithms/tiledb/module.py

```diff
@@ -109,29 +111,30 @@ def __str__(self):
         return 'TileDBFlat()'
 
 class TileDBVamana(TileDB):
-    def __init__(self, metric, _):
+    def __init__(self, metric, l_build_and_r_max_degree):
         super().__init__(
             index_type="VAMANA",
-            metric=metric
+            metric=metric,
+            l_build=l_build_and_r_max_degree,
```

Comment:
Why do we pick the same value for these 2?

Reply:
No particular reason other than I found we need to set these both to something low enough to not get stuck in the trap shown here where we hit a point where ingestion takes much longer:

![sift1m_ingestion_time_vs_accuracy](https://github.com/user-attachments/assets/974af13a-4773-4a14-9bbe-0d3f062c75d9)

But once that issue is understood and fixed I think there will be some tuning we can do to choose nicer values, though to start this seems okay.